plugins/chat/app/models/chat/message.rb

```diff
@@ -146,6 +146,12 @@ def rich_excerpt(max_length: 50)
       PrettyText.excerpt(cooked, max_length)
     end
 
+    def censored_excerpt(rich: false, max_length: 50)
+      WordWatcher.censor(
+        rich ? rich_excerpt(max_length: max_length) : excerpt(max_length: max_length),
+      )
+    end
```

Comment:
Can you expand a little it on this please?

Reply:
Yes we were doing `WordWatcher.censor` on many different serializers which was just another thing to remember, and currently only in one case (the thread list serializer) do we want the rich text excerpt, which goes back to those things we were discussing about why in reply to / edit couldn't show the proper excerpt. At some point we can just get rid of the rich param.